tox.ini

```diff
@@ -53,7 +53,7 @@ deps =
     urllib3
     pytest
     pytest-asyncio
-    pytest-operator
+    pytest-operator==1.0.0b1
```

Comment:
OH!!  How'd this do?

Reply:
well it didnt break anything so thats good!

Reply:
glad to know it works.  I'm ok with not pinning to a beta package though.  You get the ⭐ for testing it -- but lets not merge it